capstone-ui/index.html

```diff
@@ -4,7 +4,8 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Elections</title>
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">    
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <script src="https://d3js.org/d3.v5.min.js"></script>
```

Comment:
What does this do exactly?

Reply:
`<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">`

I don't know why it says this is being added, it's been here since the beginning, but Material Icons from Google provides a collection of icons that look good and can easily be used in a website. Things like arrows, plus signs, subtraction signs, and other icons. I'm not using it right now for anything so I'll remove it and add it back if I need it later.

`<script src="https://d3js.org/d3.v5.min.js"></script>`

This imports the D3.js library for the data visualization. With this, I can use the D3.js library in my JavaScript code to create the interactive map.